integration-tests/const.py

```diff
@@ -15,6 +15,7 @@
 NS = {'d': 'http://maven.apache.org/POM/4.0.0'}
 ZIP_FILE_EXTENSION = ".zip"
 CARBON_NAME = "carbon.zip"
+WSO2SERVER = 'bin/wso2server.sh'
```

Comment:
Since we have two spaces between `=` character and also we have used double-quotes in other constants declarations, shall we do the same here (to maintain a single formatting)?